### PR TITLE
Add Django and Flash playlists to `free-courses-ko.md`

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
Added
Django and Flask

### CODE REVIEW

**Comment by GH05T-HUNTER5** [comment]
> Added Django and Flask

Please Update Tamil , Malayalam , English , Korean

File: courses/free-courses-ko.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 * [Android](#android)
 * [Arduino](#arduino)
 * [ASP.NET](#asp.net)
-* [C/C++](#c/c++)
```
Comment (eshellman) [inline courses/free-courses-ko.md:7]: good catch - however, I think the best thing to do is to observe that the id that the markdown processor creates for this heading is just #cc (it removes characters not allowed in html ids and changes to lower case.)

File: courses/free-courses-ko.md [hunk 2]
```diff
@@ -39,7 +41,7 @@
 * [Windows](#windows)
 
 
-### Algorithms & Data Structures
+### <a id="algorithms-&-data-structures"></a>Algorithms & Data Structures
```
Comment (eshellman) [inline courses/free-courses-ko.md:44]: `&` is not allowed. The link in the index should be changed to `#algorithms--data-structures`

File: courses/free-courses-ko.md [hunk 3]
```diff
@@ -62,12 +64,12 @@
...
 
 
-### ASP.NET
+### <a id="asp.net"></a>ASP.NET
 
 * [ASP.NET MVC 강좌](https://www.youtube.com/playlist?list=PLbPz1r_wDPhEcKDJbOBw_3h5c2gtyDicX)
 
 
-### C/C++
+### <a id="c-and-cpp"></a>C/C++
```
Comment (eshellman) [inline courses/free-courses-ko.md:72]: change the index link instead

File: courses/free-courses-ko.md [hunk 4]
```diff
@@ -153,12 +155,12 @@
 * [AngularJS](https://www.youtube.com/playlist?list=PLs_XsVQJKaBk_JN5RctLmmVrGwEzpzqaj) - 양재동 코드랩 (YouTube)
 
 
-#### Node.js
+#### <a id="node.js"></a>Node.js
```
Comment (eshellman) [inline courses/free-courses-ko.md:158]: change the index link to `#nodejs` instead

File: courses/free-courses-ko.md [hunk 5]
```diff
@@ -153,12 +155,12 @@
...
 
 * [모던 웹을 위한 Node.js 프로그래밍](https://www.youtube.com/playlist?list=PL9kzervdzKxy-c7UQgZbnZIr14DCPM87j)
 
 
-#### React.js
+#### <a id="react.js"></a>React.js
```
Comment (eshellman) [inline courses/free-courses-ko.md:163]: Change the index link to `#reactjs` instead

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-ko.md (hunk 1 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-ko.md:7]: Ok 😃

File: courses/free-courses-ko.md (hunk 5 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-ko.md:163]: Ok

File: courses/free-courses-ko.md (hunk 3 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-ko.md:72]: Ok

File: courses/free-courses-ko.md (hunk 2 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-ko.md:44]: Ok

File: courses/free-courses-ko.md [hunk 6]
```diff
@@ -62,7 +64,7 @@
 * [아두이노 기초](https://www.youtube.com/playlist?list=PL0Vl139pNHbe-JlsydLg3NFRk6nC_cC7w)
 
 
-### ASP.NET
+### <a id="asp.net"></a>ASP.NET
```
Comment (eshellman) [inline courses/free-courses-ko.md:67]: instead, change index link to `#aspnet`

**Review by eshellman — CHANGES_REQUESTED** [review]
One more fix - sorry I missed it

File: courses/free-courses-ko.md (hunk 6 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-ko.md:67]: @eshellman 
Let me close these

File: courses/free-courses-ko.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

File: courses/free-courses-ko.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

File: courses/free-courses-ko.md (hunk 3 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

File: courses/free-courses-ko.md (hunk 4 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

File: courses/free-courses-ko.md (hunk 5 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

File: courses/free-courses-ko.md (hunk 1 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-ko.md]: > Reverted in [f7896fa](https://github.com/EbookFoundation/free-programming-books/commit/f7896fab0f35ec244ebb5b7168f93e163b6745fc)

Ok

File: courses/free-courses-ko.md (hunk 6 above)
Comment (davorpa) [reply courses/free-courses-ko.md:67]: resolved in 0f17bd78a3fdedd83b651eee6200d6b5b2abd67b

File: courses/free-courses-ko.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ### Index
 
-* [Algorithms & Data Structures](#algorithms-&-data-structures)
+* [Algorithms & Data Structures](#algorithms--data-structures)
```
Comment (davorpa) [inline courses/free-courses-ko.md:3]: Please, revert! This is addressed at #7029 

```suggestion
* [Algorithms & Data Structures](#algorithms-&-data-structures)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-ko.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Resolved at #7029 with commit 376569a55dc09ea77569298d4ffa88c7b9ee39e9

File: courses/free-courses-ko.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-ko.md]: Resolved at #7029 with commit 0f17bd78a3fdedd83b651eee6200d6b5b2abd67b

**Review by eshellman — APPROVED** [review]

**Pull request merged**